frontend/src/lib/utils/eventUsageLogic.ts

```diff
@@ -36,6 +37,18 @@ export enum DashboardEventSource {
     DashboardsList = 'dashboards_list',
 }
 
+function flattenProperties(properties: PropertyFilter[]): string[] {
+    const output = []
+    for (const prop of properties || []) {
+        if (isPostHogProp(prop.key)) {
+            output.push(prop.key)
+        } else {
+            output.push('redacted') // Custom property names are not reported
```

Comment:
Instead of multiple entries of 'redacted' in this list, do you think it would be more useful to surface the # of custom properties used in the query?